.github/workflows/docker.yml

```diff
@@ -100,14 +100,12 @@ jobs:
           image-name: server
 
       - name: Run Trivy vulnerability scanner on Admin Tools image
-        if: ${{ github.event_name == 'push' && !env.ACT }}
```

Comment:
Just verifying that this is intentional before merging.

Reply:
Yup, this fixed an issue @jackdawm and I saw where the first container would scan, the next two container scans were kipped. We can circle back to this in the future if needed!